cegs_portal/search/helpers/options.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def is_bed6(options):
```

Comment:
this doesn't need to be registered as a filter, and I think it could be used in non_targeting_reos.py